tests/test_analysis.py

```diff
@@ -1412,3 +1413,94 @@ def test_analysis_upload_file_makes_expected_calls_s3(
         [mock.call({"upload_type": "s3", "other_info": "test_value"}, "test_tag", fp)]
     )
     analysis._client._azure_upload.assert_has_calls([])
+
+
+def test_is_retraining_not_retraining(analysis_data, patched_fero_client):
+    """Test that is_retraining() returns false if the analysis is not training"""
+
+    patched_fero_client.get.return_value = analysis_data.copy()
+    analysis_data["latest_revision_model_state"] = "T"
+    analysis = Analysis(patched_fero_client, analysis_data)
+    assert analysis.is_retraining() is False
+    patched_fero_client.get.assert_called_with(
+        f"/api/analyses/{analysis_data['uuid']}/"
+    )
+
+
+def test_is_retraining_retraining(analysis_data, patched_fero_client):
+    """Test that is_retraining() returns false if the analysis is training"""
```

Comment:
This should read `is_retraining() returns true` I think.

Reply:
yes.